### PR TITLE
Adding continuous integration workflow to our package

### PR DESCRIPTION
**Pull request by jraza19**
#21

### CODE REVIEW

File: .github/workflows/R-CMD-check.yaml [hunk 1]
```diff
@@ -0,0 +1,86 @@
+# For help debugging build failures open an issue on the RStudio community with the 'github-actions' tag.
+# https://community.rstudio.com/new-topic?category=Package%20development&tags=github-actions
+on:
+  push:
+    branches:
+      - main
+      - master
```
Comment (tdkhanhvu) [inline .github/workflows/R-CMD-check.yaml]: Should we remove the master branch from here?

File: .github/workflows/R-CMD-check.yaml [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+  pull_request:
+    branches:
+      - main
+      - master
```
Comment (tdkhanhvu) [inline .github/workflows/R-CMD-check.yaml]: Should we remove the master branch from here?

File: .github/workflows/test-coverage.yaml [hunk 3]
```diff
@@ -0,0 +1,48 @@
+on:
+  push:
+    branches:
+      - main
+      - master
```
Comment (tdkhanhvu) [inline .github/workflows/test-coverage.yaml]: Should we remove the master branch from here?

File: .github/workflows/test-coverage.yaml [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+  pull_request:
+    branches:
+      - main
+      - master
```
Comment (tdkhanhvu) [inline .github/workflows/test-coverage.yaml]: Should we remove the master branch from here?

**Review by tdkhanhvu — APPROVED** [review]

File: .github/workflows/test-coverage.yaml (hunk 4 above)
Comment (jraza19) [reply .github/workflows/test-coverage.yaml]: Didn't think it hurt but i have removed it now!

**Review by jianructose — APPROVED** [review]
✔️

**Review by yhchen20 — APPROVED** [review]

**Pull request merged**